test/fixtures/project-cannot-read-files/cannot-read-1.js

```diff
@@ -0,0 +1 @@
+This file cannot be read.
```

Comment:
This might end up breaking certain kind of tooling.

Would it be possible for you to generate these files dynamically?

Reply:
In later commits, I've mocked out `fs` so that the `project-cannot-read-files` directory is no longer used at all.